guides/source/generators.md

```diff
@@ -215,10 +215,11 @@ config.generators do |g|
   g.orm             :active_record
   g.template_engine :erb
   g.test_framework  :test_unit, fixture: false
+  g.stylesheets     false
```

Comment:
This was intentionally removed in https://github.com/rails/rails/commit/d5b9618da1ac60c674e8a27c3ab4e33742e9aa9b#diff-35420b0ec9d9ea5df03a3502ff68a23ed9c60ee2938a12a978bf75a9ed8264e7L231.  See https://github.com/rails/rails/pull/44490#discussion_r811368031.